sdk/samples/iot/README.md

```diff
@@ -50,22 +99,36 @@ This section provides an overview of the different samples available to run and
 
   This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_hub_c2d_sample.c) receives incoming cloud-to-device (C2D) messages sent from the Azure IoT Hub to the device. It will successfully receive up to 5 messages sent from the service. If a timeout occurs while waiting for a message, the sample will exit. X509 authentication is used.
 
-  To send a C2D message, select your device's Message to Device tab in the Azure Portal for your IoT Hub. Enter a message in the Message Body and select Send Message.
+  <details><summary><i>How to interact with the C2D sample:</i></summary>
```

Comment:
I'm not sure to be honest.  I don't think these are going to end up on doxygen, but that's a good question.  @danewalton any thoughts on this? Currently, they really help condense the readmes so that they are easier to sift through.

Reply:
I think these sample READMEs should be part of the concept tab in doxygen:
https://azuresdkdocs.blob.core.windows.net/$web/c/docs/1.0.0-preview.5/pages.html

Reply:
Asked team about sample reademe's in doxygen and someone pointed out doxygen is only intended for the API header docs.  Some of the readmes on the link above don't add anything, so checking what people think on C sdk team about removing them?

Reply:
>  Some of the readmes on the link above don't add anything

What do you mean by this?

Reply:
IoT Hub Client and IoT Hub Device Provisioning are pretty bare.  Maybe those Readmes haven't been updated.